src/simplewallet/simplewallet.cpp

```diff
@@ -1119,11 +1162,48 @@ bool simple_wallet::sign_multisig(const std::vector<std::string> &args)
   uint32_t signers = 0;
   try
   {
-    bool r = m_wallet->sign_multisig_tx_from_file(filename, txids, [&](const tools::wallet2::multisig_tx_set &tx){ signers = tx.m_signers.size(); return accept_loaded_tx(tx); });
-    if (!r)
+    if (by_mms)
```

Comment:
I think this is matched by the `}` on line 1199. But it kind of hard to read with 2-space identations and a long if bracket. I think inverting the check - `if (!by_mms) { ...stuff ... return true; }` will make it more readable - I don't need an editor to bounce me to the other bracket.